scala-package/core/src/main/scala/org/apache/mxnet/SymbolAPI.scala

```diff
@@ -16,11 +16,19 @@
  */
 package org.apache.mxnet
 
+import scala.collection.mutable
+
 
 @AddSymbolAPIs(false)
 /**
   * typesafe Symbol API: Symbol.api._
   * Main code will be generated during compile time through Macros
   */
 object SymbolAPI extends SymbolAPIBase {
+  def Custom (op_type : String, kwargs : mutable.Map[String, Any],
+             name : String = null, attr : Map[String, String] = null) : Symbol = {
```

Comment:
should your Custom Op also take optional Seq() for positional arguments?

Reply:
Currently, `kwargs` should be able to maintain all customer needs to create a custom Op.